win32ss/gdi/ntgdi/freetype.c

```diff
@@ -48,7 +48,6 @@ extern const MATRIX gmxWorldToPageDefault;
 #define gmxWorldToDeviceDefault gmxWorldToPageDefault
 
 FT_Library  library;
-static const WORD gusEnglishUS = MAKELANGID(LANG_ENGLISH, SUBLANG_ENGLISH_US);
```

Comment:
it probably got deprecated but forgot to remove it in one of the many font rewrites.
lgtm